src/includes/classes/MenuPageOptions.php

```diff
@@ -1171,8 +1169,51 @@ public function __construct()
             echo '   </div>'."\n";
             echo '</div>'."\n";
         }
+        /* ----------------------------------------------------------------------------------------- */
+
+        if (IS_PRO || $this->plugin->isProPreview()) {
+            echo '<div class="plugin-menu-page-panel'.(!IS_PRO ? ' pro-preview' : '').'">'."\n";
+
+            echo '   <a href="#" class="plugin-menu-page-panel-heading" data-pro-version-only="'.(!IS_PRO ? __('pro version only', SLUG_TD) : '').'">'."\n";
+            echo '      <i class="si si-tablet"></i> '.__('Mobile-Adaptive Mode', SLUG_TD)."\n";
```

Comment:
"Mobile-Adaptive Mode" is a mouthful and is a bit confusing and 'unwelcoming' as a title. I suggest we use the more common name of simply "Mobile Mode" in the title and then explain "Mobile-Adaptive Mode" elsewhere inside the option panel.

In short, I suggest changing this title to "Mobile Mode" to make it easier to find and easier to identify as "the area where you configure things related to mobile devices/browsing + caching". Everything else in the panel and in the source code can stay the same. 

![2016-12-12_21-52-38](https://cloud.githubusercontent.com/assets/53005/21125709/9c31250e-c0b5-11e6-97ce-9a157aca53f8.png)